Fix searchbox alignment in firefox and chrome browsers

valerycodes opened the pull request:
### Description
Option box, search box and search button are now inline.

Fixes #350 

### Type of Change:
- User Interface

### Screenshots :
#### Firefox View:
![firefox_search](https://user-images.githubusercontent.com/11312834/36319188-6c144762-1342-11e8-8507-9805cc5aa2de.png)

#### Chrome View:
![chrome_search](https://user-images.githubusercontent.com/11312834/36319233-8e346f02-1342-11e8-9fd1-8c9ece00226a.png)

#### Mobile View:
![mobile_view](https://user-images.githubusercontent.com/11312834/36319250-9ebd4074-1342-11e8-9bd8-515ef71c66a6.png)

valerycodes committed:
Fix searchbox alignment in firefox and chrome browsers

medhach commented on templates/ui/dashboard.html:
```diff
@@ -50,16 +50,14 @@ <h3>
 					<center>
 						<p>...</p>
-				
+
```
You can remove this change

medhach commented on templates/ui/dashboard.html:
```diff
@@ -50,16 +50,14 @@ <h3>
 							<a href="javascript:void(0)" onclick="window.location.href='/pcsa/list_posts/'"class="btn btn-primary">Go to FirstAide</a>
 					</center>
-				</div>	
+				</div>
```
Same. Please remove this change.

medhach commented on templates/ui/dashboard.html:
```diff
@@ -34,8 +34,8 @@ <h3>
-					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.		
-					<center>	
+					The Malaria Mobile App was developed to help keep Peace Corps volunteers safe in places that are at a high risk of acquiring malaria.
+					<center>
```
Please remove this change. Make sure you don't commit empty changes next time.

valerycodes replied:
I didn't do that formating and I don't know why it was formatted that way. Doing anything formatting will make that section of code unreadable

medhach commented:
@valerycodes Please make the requested changes and amend to the previous commit. :)

valerycodes committed:
Update changes

The for lines, 38 and 56 the was no white space to be remove as indicated by my text editor.

medhach commented:
@valerycodes Sorry for commenting so late about this, but can you also confirm if the change is working well on IE and Edge explorers?

valerycodes commented:
@medhach Sorry, I don't use windows OS and I don't have it install on my machine. But will work on any browser that supports Bootsrap 3
